### PR TITLE
295396 update install adapter section

### PR DESCRIPTION
**Pull request by osisoft-mbishop**

### CODE REVIEW

File: content/main/shared-content/_includes/README [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+--|--|--
+component-id.md | Instance-specific component ID used during setup | `BACnet1`
+component-type.md | Adapter-specific type used during setup | `BACnet`
+docker-image.md | Name of adapter-specific docker image file | `bacnetdockerstart.sh`
+framework-version.md | Version of framework that adapter uses | `1.4`
+installer-name.md | Name of the adapter-specific installer. Need to manually add installer version and architecture | `PI-Adapter-for-BACnet`
+product-name.md | What it sounds like. | `PI Adapter for BACnet`
+product-version | Product version | `1.1`
+startup-script | Script to startup docker-image | `eventhubsdockerstart.sh`
+symantic-version | Full symantic version of the product | `1.1.1.1`
```
Comment (rmuolic) [inline content/main/shared-content/_includes/README:15]:
```suggestion
product-version.md | Product version | `1.1`
startup-script.md | Script to startup docker-image | `eventhubsdockerstart.sh`
symantic-version.md | Full symantic version of the product | `1.1.1.1`
```

File: content/main/shared-content/installation/install-the-adapter.md [hunk 2]
```diff
@@ -30,17 +30,19 @@ Complete the following steps to install a PI adapter on a Windows computer:
 
 ## Linux
 
-Complete the following steps to install a PI adapter on a Linux computer:
+Complete the following steps to install an adapter on a Linux computer:
 
-1. Download the appropriate Linux distribution file from the [OSIsoft Customer portal (https://customers.osisoft.com/s/products)](https://customers.osisoft.com/s/products).
+1. Download the appropriate Linux distribution file (<code>[!include[installer](../_includes/inline/installer-name.md)]-{PLATFORM}_.deb</code>) from the [OSIsoft Customer portal](https://customers.osisoft.com/s/products).
```
Comment (rmuolic) [inline content/main/shared-content/installation/install-the-adapter.md:35]: Is {PLATFORM} correct here?

File: content/main/shared-content/installation/install-the-adapter.md [hunk 3]
```diff
@@ -30,17 +30,19 @@ Complete the following steps to install a PI adapter on a Windows computer:
...
 
     **Note:** Customer login credentials are required to access the portal.
 
 2. Open a terminal.
 
 3. Run the `sudo apt update` command to update available packages information.
 
-4. Run the `sudo apt install` command against the Linux distribution file selected in step 1 of this procedure.
+4. Run the `sudo apt install` command against the Linux distribution file (<code>[!include[installer](../_includes/inline/installer-name.md)]-{PLATFORM}_.deb</code>) selected in step 1 of this procedure.
```
Comment (rmuolic) [inline content/main/shared-content/installation/install-the-adapter.md:43]: Is {PLATFORM} correct here? In DNP3, {ARC} is used instead.

**Review by rmuolic — COMMENTED** [review]
Just unsure about {PLATFORM}.

File: content/main/shared-content/installation/install-the-adapter.md (hunk 2 above)
Comment (osisoft-mbishop) [reply content/main/shared-content/installation/install-the-adapter.md]: updated to _platform_ per MS styleguide

File: content/main/shared-content/installation/install-the-adapter.md (hunk 3 above)
Comment (osisoft-mbishop) [reply content/main/shared-content/installation/install-the-adapter.md]: updated to _platform_ per MS styleguide

**Review by rmuolic — APPROVED** [review]

**Pull request merged**